apps/console/src/features/roles/pages/create-role-wizard.tsx

```diff
@@ -96,19 +99,27 @@ const CreateRoleWizard: FunctionComponent<CreateRoleProps> = (props: CreateRoleP
                 )) || [];
     
             const roleData: CreateRoleInterface = {
-                audience: roleAudience === RoleAudienceTypes.ORGANIZATION
+                displayName: stepperState[ CreateRoleStepsFormTypes.BASIC_DETAILS ].roleName,
+                permissions: selectedPermissionsList,
+                schemas: []
+            };
+
+            // If the organization is a super or first level organization, no need to send the audience.
+            const isSuperOrFirstLevelOrg: boolean = [
+                OrganizationType.SUPER_ORGANIZATION,
+                OrganizationType.FIRST_LEVEL_ORGANIZATION
+            ].includes(orgType);
```

Comment:
Shall we move this to the util file ?

Reply:
Improved the already available `useGetOrganizationType` hook with https://github.com/wso2/identity-apps/pull/4522/commits/2052240885356625ba7e06fa6fb4860a9132394c

Reply:
@nipunsampath FYI

Reply:
Ack